lib/v2/datasource/remote/api/network_repository.dart

```diff
@@ -29,7 +28,7 @@ abstract class NetworkRepository {
       case 200:
         {
           print('Model Class: ' + modelMapper.toString());
-          var body = json.decode(response.body);
+          var body = response.parseJson();
```

Comment:
Please move the `response_extension.dart'` to v2 code before using it.
And make sure this didnt break other places in v2. 

Thanks.

Reply:
I know it was a uft8 decoding thing.

Reply:
I can move the file to v2 sure

As for "does it break other things" - this is not a change, it's a fix. It worked this way the entire time in V1. So confident it will work because it worked this way in V1. 

The change was to do it different than V1, which caused the bug.

Reply:
File moved to v2